2024-01-03 03:11:58,219 (7fa16ad95808) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: 871adbd (Tue Oct 24 14:29:29 UTC 2023)
2024-01-03 03:11:58,219 (7fa16ad95808) :  DEBUG (core:361) - Using the elevated policy
2024-01-03 03:11:58,220 (7fa16ad95808) :  DEBUG (core:450) - Starting runtime component.
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (core:450) - Starting caching component.
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (core:450) - Starting data component.
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (core:450) - Starting networking component.
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (networking:262) - Loaded HTTP cookies
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (networking:434) - Setting the default network timeout to 20.0
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (core:450) - Starting localization component.
2024-01-03 03:11:58,221 (7fa16ad95808) :  INFO (localization:409) - Setting the default locale to en-us
2024-01-03 03:11:58,221 (7fa16ad95808) :  DEBUG (core:450) - Starting messaging component.
2024-01-03 03:11:58,222 (7fa16ad95808) :  DEBUG (core:450) - Starting debugging component.
2024-01-03 03:11:58,222 (7fa16ad95808) :  DEBUG (core:450) - Starting services component.
2024-01-03 03:11:58,222 (7fa167950b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/system/messaging/clear_events/com.plexapp.agents.imdb'
2024-01-03 03:11:58,222 (7fa16ad95808) :  DEBUG (core:450) - Starting myplex component.
2024-01-03 03:11:58,223 (7fa16ad95808) :  DEBUG (core:450) - Starting notifications component.
2024-01-03 03:11:58,286 (7fa16ad95808) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.agents.imdb in namespace 'metadata'
2024-01-03 03:11:58,288 (7fa16ad95808) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-01-03 03:11:58,386 (7fa16ad95808) :  ERROR (networking:197) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-01-03 03:11:58,388 (7fa16ad95808) :  CRITICAL (runtime:1299) - Exception getting hosted resource hashes (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-871adbd44/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1293, in get_resource_hashes
    json = self._core.networking.http_request("http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes", timeout=10).content
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-871adbd44/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 243, in content
    return self.__str__()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-871adbd44/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 221, in __str__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-871adbd44/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 159, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

2024-01-03 03:11:58,392 (7fa167731b38) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2024-01-03 03:11:58,392 (7fa16ad95808) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2024-01-03 03:11:58,392 (7fa167731b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2024-01-03 03:11:58,393 (7fa16ad95808) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2024-01-03 03:11:58,393 (7fa1676ebb38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400'
2024-01-03 03:11:58,393 (7fa16ad95808) :  DEBUG (core:150) - Finished starting framework core
2024-01-03 03:11:58,394 (7fa16ad95808) :  DEBUG (core:560) - Loading plug-in code
2024-01-03 03:11:58,414 (7fa1676ebb38) :  DEBUG (core:538) - Machine identifier is 4aa082f4f7d6dd1a83a571d03b236b07794db03c
2024-01-03 03:11:58,414 (7fa167731b38) :  DEBUG (services:362) - Loaded services
2024-01-03 03:11:58,415 (7fa1676ebb38) :  DEBUG (core:539) - Server version is 1.32.7.7621-871adbd44
2024-01-03 03:11:58,418 (7fa16770eb38) :  DEBUG (services:438) - No shared code to load
